workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/exception/FileServerUnavailableException.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.customer.exception;
+
+/**
+ * @author Karoline Silva
+ */
+public class FileServerUnavailableException extends RuntimeException {
```

Comment:
I think all of these should extend a regular Exception
https://stackoverflow.com/questions/4233793/proper-use-of-runtimeexception